flink-runtime/src/main/java/org/apache/flink/runtime/jobmaster/JobMasterGateway.java

```diff
@@ -55,6 +55,14 @@
 public interface JobMasterGateway extends CheckpointCoordinatorGateway, FencedRpcGateway<JobMasterId> {
 
 	/**
+	 * Cancel the currently executed job.
```

Comment:
it would be more consistent with the javadoc of other method to say "Cancels"

Reply:
Hmm, not sure if that is true tbh. We actually have quite a wild mix of how we write JavaDocs in `JobMasterGateway`. I will change all of them starting with a verb in the 3rd person.